docs/android/emulator/hardware-acceleration.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Enable hardware acceleration for the Android emulator (Hyper-V & HAXM)"
+description: "Learn how to enable hardware acceleration features to maximize Android emulator performance for a .NET MAUI app."
+ms.date: 01/19/2022
+ms.topic: how-to
+---
+
+# How to enable hardware acceleration with Android emulators (Hyper-V & HAXM)
+
+This article explains how to use your computer's hardware acceleration features to maximize Android emulator performance.
+
+With Visual Studio, you can easily test and debug your .NET MAUI app for Android in situations where an Android device isn't available. However, if hardware acceleration isn't available or enabled, the emulator will run too slow. You can drastically improve the performance of the emulator by enabling hardware acceleration and using special x86 virtual device images.
```

Comment:
Should all mentions of the device images in this article be saying they should be x86_64?

Reply:
I was confused, I mean it's the default. But the original article states what is written here: x86 is the best image for speed? The guidance from Xamarin articles seem to say stay away from x64. Are they just old and super out of date?

Reply:
Yes. Any content under /ios or /android was handed to the product team back in 2018 and hence isn't up to date.

The x86 image might be faster (I honestly don't know) but the requirement to submit 64 bit versions of apps to the Play store has been in place since 2019.